.github/workflows/tests.yml

```
@@ -123,6 +125,13 @@ jobs:
         env:
           PYTHON_VERSION: ${{ matrix.python }}  # Why do this?
       - uses: seanmiddleditch/gha-setup-ninja@v6
+      - name: Install wasi-sdk (Windows)
+        shell: pwsh
+        if: ${{ startsWith(matrix.os, 'windows') }}
```

```suggestion
        if: runner.os == 'Windows'
```
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#detecting-the-operating-system